tests/unit/layout-test.js

```diff
@@ -163,30 +162,30 @@ describe('layout', function () {
     g.setNode('b', {width: 100, height: 100})
     g.setEdge('a', 'b', {minlen: 2})
     layout(g)
-    var points = g.edge('a', 'b').points
+    let points = g.edge('a', 'b').points
     expect(points).to.have.length(5)
     expect(points).eqls([
-      {x: 100 / 2, y: 100},           // intersect with bottom of a
+      {x: 100 / 2, y: 100}, // intersect with bottom of a
       {x: 100 / 2, y: 100 + 200 / 2}, // bend #1
       {x: 100 / 2, y: 100 + 400 / 2}, // point for edge label
       {x: 100 / 2, y: 100 + 600 / 2}, // bend #2
-      {x: 100 / 2, y: 100 + 800 / 2}  // intersect with top of b
+      {x: 100 / 2, y: 100 + 800 / 2} // intersect with top of b
     ])
   })
 
   // FIXME: get tests passing
-  describe.skip('can layout a self loop', function () {
-    _.forEach(['TB', 'BT', 'LR', 'RL'], function (rankdir) {
+  describe('can layout a self loop', function () {
```

Comment:
Did you get this test passing or is this from your automated linting tools? If you did we can remove this comment.

Reply:
I did get it passing. I will remove comment.